components/experimental/src/measure/provider/single_unit.rs

```diff
@@ -0,0 +1,28 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use super::si_prefix::SiPrefix;
+
+/// Represents a single unit in a measure unit.
+/// For example, the MeasureUnit `kilometer-per-square-second` contains two single units:
+///    1. `kilometer` with power 1 and prefix 3 with base 10.
+///    2. `second` with power -2 and prefix `NotExist`.
```

Comment:
how can the prefix be `NotExist` if the `si_prefix` field is not optional?

Reply:
not exist means `Decimal` or `Binary` with  a power equal to `0`

We can say, and prefix power equal `0`